src/generic_ui/scripts/user.ts

```diff
@@ -41,46 +40,61 @@ module UI {
       }
       this.name = profile.name;
       this.imageData = profile.imageData || UI.DEFAULT_USER_IMG;
-      this.isOnline = profile.isOnline;
     }
 
     // Returns two strings, where each matches an array name in model.contacts.
     // Does not use an enum because we need a string value, and typescript
     // enums evaluate to numbers.
```

Comment:
Yes, using constant names values is fine too. 

There are basically two problems with this code at the moment (which I'm fine with adding TODOs with for now as it's not part of this pull request): 
 1. explicit use of strings for constant values
 2. the implicit dependency between a constructed string value and the naming of a field in another javascript object (the  model ui in this case).

Enums/constants tackle part of this problem (1), but neither addresses (2). 

I've filed an issue here for continued exploration of what we should do about this: https://github.com/uProxy/uproxy/issues/769

Feel free to put in a `CONSIDER: https://github.com/uProxy/uproxy/issues/769` for now.